src/utils.ts

```diff
@@ -76,3 +76,9 @@ export function addDays(date: Date, days: number) {
   newDate.setDate(date.getDate() + days);
   return newDate;
 }
+
+export function getStartOfDate(date: Date) {
+  return typeof date.getMonth === 'function'
+    ? new Date(date.getFullYear(), date.getMonth(), date.getDate())
+    : '';
```

Comment:
Mmm, I am not sure about it. 

I would prefer returning `undefined`, `null` or throwing an exception.

Reply:
fixed, i return and undefined